runtime/cudaq/utils/extension_point.h

```diff
@@ -33,47 +33,47 @@ namespace cudaq {
 ///    This class should declare pure virtual methods that extensions will
 ///    implement.
 ///
-/// @code
+/// <code>
 /// class MyExtensionPoint : public cudaq::extension_point<MyExtensionPoint> {
 /// public:
-///   virtual std::string parrotBack(const std::string &msg) const = 0;
+///   virtual std::string parrotBack(const std::string &) const = 0;
```

Comment:
It looks like spell check is failing on this. Why? It's clearly in a code block?

Reply:
looks like pyspelling does not ignore the <code> blocks correctly, I tied to play with configuration to no avail, so I changed them to markdown quotes (```) instead